todo_app/tests/test_items.py

```diff
@@ -1,88 +1,88 @@
-import Trello_items, pytest, datetime
+import Mongo_items, pytest, datetime
```

Comment:
These unit tests are passing with a bit of luck in Travis - they actually only pass on some days of the year - and this is because of a real application bug, proving their use! See my comments below on the ViewModel